mesh/block.go

```diff
@@ -6,14 +6,25 @@ import (
 	"github.com/davecgh/go-xdr/xdr2"
 	"github.com/google/uuid"
 	"github.com/spacemeshos/go-spacemesh/address"
-	"io"
 	"math/big"
 	"time"
 )
 
 type BlockID uint64
-
 type LayerID uint64
+type TransactionId []byte
+
+type BlockHeader struct {
+	Id         BlockID
+	LayerIndex LayerID
+	MinerID    string
+	Data       []byte
+	Coin       bool
+	Timestamp  int64
+	BlockVotes []BlockID
+	ViewEdges  []BlockID
+	TxIds      []TransactionId
+}
 
 type Block struct {
```

Comment:
Why not embed BlockHeader into block?

Reply:
maybe later on. right now this only concerns meshDB so no reason